services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_scheduler.py

```diff
@@ -135,9 +135,9 @@ async def get_service_state(
     ],
 ):
     # pylint: disable=protected-access
-    return dynamic_sidecars_scheduler._scheduler.get_scheduler_data(
+    return dynamic_sidecars_scheduler._scheduler.get_scheduler_data(  # noqa: SLF001
```

Comment:
Question: what about making that thing accessible instead of making it private and still accessing it??